hw6/task3.py

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
Question 3: (8/15)

Well done on obtaining the correct result - remember that a function needs to perform a task - not print certain lines. The best way to go about this would be to use a dictionary that would store all the URLs and choices available for the pages.

A dictionary was not used to store links. (-5)
No time or space complexity is included. (-2)

Time and space complexity would be O(1) as there is no increment within the loop.